src/Storage/Storage.Management/Az.Storage.psd1

```diff
@@ -57,7 +57,17 @@ RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '1.3.0'; })
 
 # Assemblies that must be loaded prior to importing this module
 RequiredAssemblies = '.\Microsoft.Azure.Management.Storage.dll', 
-               '.\Microsoft.WindowsAzure.Storage.dll'
+               '.\Microsoft.Azure.Storage.Common.dll',
+               '.\Microsoft.Azure.Storage.Blob.dll',
+               '.\Microsoft.Azure.Storage.File.dll',
+               '.\Microsoft.Azure.Storage.Queue.dll',
+               '.\Microsoft.Azure.Cosmos.Table.dll',
+               '.\Microsoft.Azure.DocumentDB.Core.dll',
+               '.\Microsoft.WindowsAzure.Storage.DataMovement.dll',
```

Comment:
We can't move the DMLib dependency

Reply:
As we discussed before, will try to change the name or add DMlib code to PSH.

Reply:
As we discussed, I have put the dmlib with new name in src/lib